exams/pearson_test.py

```diff
@@ -224,6 +224,25 @@ def test_profile_country_to_alpha3_invalid_country(self):  # pylint: disable=no-
         with self.assertRaises(InvalidProfileDataException):
             CDDWriter.profile_country_to_alpha3(profile)
 
+    def test_profile_phone_number_functions(self):
+        """
+        A profile with an invalid phone number should raise an InvalidProfileDataException
+        """
+        for case in ['', None, 'bad string', '120272727']:
```

Comment:
Data Driven Tests, it's a python module we use to parameterize tests on different data sets. It also tries to give each run of the test a meaningful name based on the data:

Docs: http://ddt.readthedocs.io/en/latest/example.html

Example usage in one of our tests: https://github.com/mitodl/micromasters/blob/master/exams/pearson_test.py#L367

In this case, you'd split the false cases into a `ddt`-based test and keep the positive test case at the end of the method in a regular test method.